pt.html

```diff
@@ -381,59 +404,59 @@
 <panel w=500 h=450>
 <pic src="pics/sci0.png" sx=0 sy=0></pic>
 <words x=10 y=10 w=400 h=60>
-	In 1885, Hermann Ebbinghaus performed an act of scientific masochism.
+	Em 1885, Hermann Ebbinghaus realizou um ato de masoquismo científico.
 </words>
 <words x=30 y=320 w=430 h=90>
-	The German psychologist memorized <i>thousands</i> of nonsense words,
-	recorded how much he forgot over time, and discovered...
+	O psicólogo alemão memorizou <i>milhares</i> de palavras sem sentido,
+	e registrou o quanto esqueceu com o tempo e descobriu...
 </words>
 </panel>
 
 <panel w=400 h=500>
 <pic src="pics/sci0.png" sx=500 sy=0></pic>
 <words x=60 y=10 w=250>
-	<b>THE FORGETTING CURVE</b>
+	<b>A CURVA DO ESQUECIMENTO</b>
 </words>
 <words x=10 y=310 w=350>
-	He found that you forget most of what you learn in the first 24 hours,
-	then – if you don’t practice recall – your remaining memories decay exponentially.*
+	Ele descobriu que você esquece a maior parte do que aprende nas primeiras 24 horas,
+	então - se você não pratica a rememoração - suas memórias restantes decaem exponencialmente. *
 </words>
 </panel>
 
 <panel w=500 h=53 style="margin-top:-5px">
 <words w=500 x=-15 y=-10 no-bg style="width: 500px; font-size:0.8em; text-align: right; color:#999; line-height: 1.1em;">
-	* technically the curve isn't <i>exactly</i>
+	* tecnicamente não é <i>exatamente</i> uma curva
 	<br>
-	exponential, but, eh, close enough.
+	exponencial, mas, bem, é perto suficiente.
 </words>
 </panel>
 
 <panel w=500 h=450>
 <pic src="pics/sci0.png" sx=900 sy=0></pic>
 <words x=10 y=10 w=450 h=90>
-	Philosophers have debated about memory for millennia,
-	but Ebbinghaus was the first to do actual <i>experiments.</i>
-	(which have been replicated)
+	Filósofos têm debatido sobre a memória por milênios,
+	mas Ebbinghaus foi o primeiro a fazer <i>experimentos</i> reais.
+	(que foram replicados)
 </words>
 <words x=193 y=169 w=70 no-bg class="comic_text" style="color:rgba(0,0,0,0.35)">
-	plz...
+	favor...
```

Comment:
No original é `plz` talvez fosse bom traduzir como abreviação também

```suggestion
	pfvr...
```

Reply:
Esse é um daqueles pontos que não estou certo de mudar ou não (também pense em deixar resumido, tipo "plz" em vez pfvr de inicio). Por isso que coloque as duas chamadas seguintes foram também "me" e "matar" para ficar de acordo.

Porém, esse ponto junto com outro antes que não atualizei, creio que pode valer a pena pelo menos mais uma pessoa pra confrmar ou reconfirmar, e talvez e talvez revisar com alguem que fale portugues não do Brasil. Ou mesmo no https://portuguese.stackexchange.com (assim como pretendo fazer com alguns outros termos mais oficiais aqui).

Escrevi tudo isso porque esse é um segundo ponto que precisa ser revisto, não que não vai ser aplicado exatamente como falou.